llvm/test/CodeGen/AIE/alias-analysis-pointer-desambiguation.ll

```diff
@@ -0,0 +1,160 @@
+;
+; This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+; See https://llvm.org/LICENSE.txt for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+;
+; (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+; RUN: opt -mtriple=aie2 -passes=aa-eval -print-all-alias-modref-info -disable-output < %s 2>&1 | FileCheck %s --check-prefix=ENABLED-AIE-AA-DES
+; RUN: opt -mtriple=aie2 -passes=aa-eval -print-all-alias-modref-info -disable-output \
+; RUN:    --aie-alias-analysis-disambiguation=false < %s 2>&1 | FileCheck %s --check-prefix=DISABLED-AIE-AA-DES
```

Comment:
DIgital Encryption Standard?

Reply:
Maybe I tried some AA improvement and got a new encryption alg instead.